.github/workflows/robot-command-dispatcher.yaml

```diff
@@ -0,0 +1,56 @@
+#
+#  Licensed to the Apache Software Foundation (ASF) under one or more
+#  contributor license agreements. See the NOTICE file distributed with
+#  this work for additional information regarding copyright ownership.
+#  The ASF licenses this file to You under the Apache License, Version 2.0
+#  (the "License"); you may not use this file except in compliance with
+#  the License. You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+name: Kroxylicious Robot Command Dispatcher
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  process-command:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.issue.pull_request }} && contains(github.event.comment.body, '@kroxylicious-robot')
+    steps:
+      - name: 'Check team membership'
+        if: ${{ github.repository == 'kroxylicious/kroxylicious' }}
+        uses: tspascoal/get-user-teams-membership@v2
+        id: team-membership
+        with:
+          username: ${{ github.actor }}
+          team: admins
```

Comment:
I'm fine with `admins` for now, but we should create a release team or grant all members of the `engineering` team release rights.

Reply:
Yes, agree.   I had the same idea.   I wanted to get some feedback on the rough direction of the PR before adding the team.  I'll do it.

Reply:
I've switch to a dedicated team (release-engineers)